domain/src/main/java/org/oppia/domain/util/StateRetriever.kt

```diff
@@ -512,10 +513,32 @@ class StateRetriever @Inject constructor(
       "TextInput" -> {
         createTextInputCustomizationArgsMap(customizationArgsJson)
       }
+      "RatioExpressionInput" -> {
+        createRatioExpressionInputCustomizationArgsMap(customizationArgsJson)
+      }
       else -> mutableMapOf()
     }
   }
 
+  private fun createRatioExpressionInputCustomizationArgsMap(
```

Comment:
Please add tests for the schema objects being parsed, too.

Reply:
done